index.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
     'func-style': [2],
     'class-methods-use-this': [2],
     'linebreak-style': [2],
-    'max-len': [2, {code: 100}],
+    'max-len': [1, {code: 100}],
```

Comment:
I do agree with the changes and rationale of this commit. Complying with the other rules, sometimes, can lead to non-trivial re-factors that we want, as developers, to address later on. However, I do think that `max-len` is quite easy to comply with, so it doesn't make sense to me to have it as a warning.

Reply:
Ah. You're right. I didn't mean to include line length; I mistook this this for the number of lines in a function.

Reply:
@dicearr updated